operator/pom.xml

```diff
@@ -33,7 +33,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <quarkus.operator.sdk.version>4.0.6</quarkus.operator.sdk.version>
-        <quarkus.version>2.13.6.Final</quarkus.version>
```

Comment:
The Quarkus version for the Operator is tied to the SDK, we cannot override the version like this, I'm afraid. This property is here just to align some additional dependencies with the SDK Quarkus version.

Reply:
@vmuzikar updated the Operator SDK to keep it in sync with Quarkus. If we don't upgrade to the next  release, I'm afraid that we won't be free of several CVEs currently reported.